vizro-core/src/vizro/static/css/container.css

```
@@ -0,0 +1,30 @@
+.container-title,
+.container-title-collapse {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0;
+}
```

We only want to remove the margin-bottom for the heading when it's collapsible :) 

```suggestion
.container-title,
.container-title-collapse {
  align-items: center;
  display: flex;
  justify-content: space-between;
}

.container-title-collapse {
  margin-bottom: 0;
}
```